### PR TITLE
(IMAGES-1139) BHG support for winfips-2012r2-x64

### PR DESCRIPTION
**Pull request by Dorin-Pleava**
Add beaker-hostgenerator support for winfips-2012r2-x64

### CODE REVIEW

**Comment by Dorin-Pleava** [comment]
PR Tests are failing because of this: [IMAGES-1144](https://tickets.puppetlabs.com/browse/IMAGES-1144)

File: lib/beaker-hostgenerator/data.rb [hunk 1]
```diff
@@ -1455,6 +1455,16 @@ def osinfo
             'template' => 'win-2012r2-x86_64'
           }
         },
+        'windows2012r2fips-64' => {
```
Comment (jcoconnor) [inline lib/beaker-hostgenerator/data.rb]: Do you need a ```windows2012r2fips-6432``` definition here as well?
Most of the other platforms include this as well.

Comment (Dorin-Pleava) [reply lib/beaker-hostgenerator/data.rb]: Added windows2012r2fips-6432 with its fixtures. Wasn't sure, but it seems it should be added.

Comment (jcoconnor) [reply lib/beaker-hostgenerator/data.rb]: 👍

**Pull request merged**